clients/lsp-yaml.el

```diff
@@ -172,14 +172,21 @@ Limited for performance reasons."
                                         (lsp-package-ensure 'yaml-language-server
                                                             callback error-callback))))
 
+(defconst lsp--yaml-schema-extension-type
+      '(list
+       (cons 'name  string)
+       (cons 'description string)
+       (cons 'url string)
+       (cons 'fileMatch (repeat string))))
```

Comment:
Can you add documentation to this variable? The indentation seems a bit off here. 🤔